src/m2kcalibration_impl.cpp

```diff
@@ -419,12 +422,14 @@ bool M2kCalibrationImpl::fine_tune(size_t span, int16_t centerVal0, int16_t cent
 		std::this_thread::sleep_for(std::chrono::milliseconds(5));
 
 		ch_data = m_m2k_adc->getSamplesRaw(num_samples);
+		m_m2k_adc->stopAcquisition();
```

Comment:
No new line after `stopAcquisition()`, in some places a new line after `stopAcquisition()` was introduced. Should be consistent.